lib/decompress/zstd_decompress_block.c

```diff
@@ -1112,69 +1175,95 @@ ZSTD_decompressSequences_body( ZSTD_DCtx* dctx,
                 BIT_DStream_endOfBuffer < BIT_DStream_completed &&
                 BIT_DStream_completed < BIT_DStream_overflow);
 
+        /* decompress without overrunning litPtr begins */
+        if (litInDst)
+        {
 #if defined(__GNUC__) && defined(__x86_64__)
-        /* Align the decompression loop to 32 + 16 bytes.
```

Comment:
This was a relatively useful and informative comment.
Unless you have reason to believe that the comment is misleading or useless,
please preserve this knowledge in the source code.

Reply:
Ah, I had intended to move it slightly, not remove it.